app/uk/gov/hmrc/gform/gformbackend/GformBackEndService.scala

```diff
@@ -217,13 +226,21 @@ class GformBackEndService(
           cache.form,
           cache.formTemplate
         )
-      instructionsRenderingService
-        .createInstructionPDFHtml(
+
+      pdfRenderService
+        .createPDFHtml[D, U, PDFType.Instruction](
+          s"Instructions PDF - ${cache.formTemplate.formName.value}",
+          None,
           cache,
-          submissionDetails,
-          formModelOpticsUpdated
+          formModelOpticsUpdated,
+          cache.formTemplate.destinations match {
+            case DestinationList(_, acknowledgementSection, _) =>
+              acknowledgementSection.instructionPdf.map(p => PDFModel.HeaderFooter(p.header, p.footer))
+            case _ => None
+          },
+          submissionDetails
         )
-        .map(Some(_))
+        .map(Option(_))
```

Comment:
Why was this changed to `Option.apply` ?

Reply:
Reverted